packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -747,6 +752,335 @@ export class ProjectModel {
         }
     }
 
+    async duplicateContent(projectUuid: string, previewProjectUuid: string) {
```

Comment:
We should add a sentry transaction here to see how long this takes for big orgs and if it impacts server health.